wps/classes/wps.listener.php

```diff
@@ -11,6 +11,21 @@ function ongetMapAdditions ($event) {
 
         $bp = jApp::config()->urlengine['basePath'];
         $js = array(
+            jUrl::get('jelix~www:getfile', array('targetmodule'=>'wps', 'file'=>'OpenLayers/Format/WCSGetCoverage.js')),
+            jUrl::get('jelix~www:getfile', array('targetmodule'=>'wps', 'file'=>'OpenLayers/Format/Filter.js')),
+            jUrl::get('jelix~www:getfile', array('targetmodule'=>'wps', 'file'=>'OpenLayers/Format/Filter/v1.js')),
+            jUrl::get('jelix~www:getfile', array('targetmodule'=>'wps', 'file'=>'OpenLayers/Format/Filter/v1_1_0.js')),
+            jUrl::get('jelix~www:getfile', array('targetmodule'=>'wps', 'file'=>'OpenLayers/Format/OWSCommon.js')),
+            jUrl::get('jelix~www:getfile', array('targetmodule'=>'wps', 'file'=>'OpenLayers/Format/OWSCommon/v1.js')),
+            jUrl::get('jelix~www:getfile', array('targetmodule'=>'wps', 'file'=>'OpenLayers/Format/OWSCommon/v1_0_0.js')),
+            jUrl::get('jelix~www:getfile', array('targetmodule'=>'wps', 'file'=>'OpenLayers/Format/OWSCommon/v1_1_0.js')),
+            jUrl::get('jelix~www:getfile', array('targetmodule'=>'wps', 'file'=>'OpenLayers/Format/WFST.js')),
+            jUrl::get('jelix~www:getfile', array('targetmodule'=>'wps', 'file'=>'OpenLayers/Format/WFST/v1.js')),
+            jUrl::get('jelix~www:getfile', array('targetmodule'=>'wps', 'file'=>'OpenLayers/Format/WFST/v1_1_0.js')),
+            jUrl::get('jelix~www:getfile', array('targetmodule'=>'wps', 'file'=>'OpenLayers/Format/WPSDescribeProcess.js')),
```

Comment:
```suggestion
            jUrl::get('jelix~www:getfile', array('targetmodule'=>'wps', 'file'=>'OpenLayers/Format/WPSDescribeProcess.js')),
            jUrl::get('jelix~www:getfile', array('targetmodule'=>'wps', 'file'=>'OpenLayers/Format/WPSDescribeProcess/v_1_0_0.js')),
```
The `WPSDescribeProcess/v_1_0_0.js` contains the functions,  `WPSDescribeProcess.js` is needed but do nothing

Reply:
I'm using an OL2 fork from `2.13.1` branch not `master` so there is not this file : 
https://github.com/3liz/ol2/tree/lizmap-build-2.13.1/lib/OpenLayers/Format
and `WPSDescribeProcess.js` does something : 
https://github.com/3liz/ol2/blob/lizmap-build-2.13.1/lib/OpenLayers/Format/WPSDescribeProcess.js